edx_proctoring/tests/test_api.py

```diff
@@ -482,6 +482,25 @@ def test_remove_exam_attempt(self):
         with self.assertRaises(StudentExamAttemptDoesNotExistsException):
             remove_exam_attempt(proctored_exam_student_attempt.id)
 
+    @ddt.data(
+        ProctoredExamStudentAttemptStatus.verified,
+        ProctoredExamStudentAttemptStatus.submitted,
+        ProctoredExamStudentAttemptStatus.error,
+    )
+    def test_remove_exam_attempt_with_status(self, to_status):
+        """
+        Test to remove the exam attempt which calls
+        the Credit Service method `remove_credit_requirement_status`.
+        """
+
+        proctored_exam_student_attempt = self._create_unstarted_exam_attempt()
+        # now update the status of the exam.
+
+        proctored_exam_student_attempt.status = to_status
+        proctored_exam_student_attempt.save()
+        # now remove exam attempt which calls the credit service method 'remove_credit_requirement_status'
+        remove_exam_attempt(proctored_exam_student_attempt.id)
+
```

Comment:
You're not asserting in the test that the requirement_status data was actually removed here :-)

I'll add it so we can merge it in (provided the corresponding change in edx-platform has been +1'd)